Week3/homework/step2-3.js

```diff
@@ -30,9 +28,13 @@ console.log('while', repeatStringNumTimesWithWhile('abc', 3));
 function repeatStringNumTimesWithDoWhile(str, num) {
   // eslint-disable-next-line prefer-const
   let result = '';
-
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
+  if (num <= 0) {
+    return result;
```

Comment:
You could also add this `if` condition to `for` and `while` loop, right?

Reply:
hmm, when thinking now I see that's possible. I'll look into it